nupic/research/frameworks/pytorch/imagenet/common_experiments.py

```diff
@@ -24,13 +24,16 @@
     ImagenetExperiment,
 )
 
-
 class RezeroedKWinnersImagenetExperiment(mixins.RezeroWeights,
                                          mixins.UpdateBoostStrength,
                                          ImagenetExperiment):
     pass
 
+class KnowledgeDistillationImagenetExperiment(mixins.KnowledgeDistillation,
```

Comment:
You don't really need to create this shared experiment class unless you find it helpful. I only created this file because Luiz pointed out that I was using the same `RezeroedKWinnersImagenetExperiment` class in a couple of places. It is fine for experiment CONFIG.py files to define their own experiment classes.